subsys/bluetooth/mesh/shell.c

```diff
@@ -3263,3 +3623,58 @@ static int cmd_mesh(const struct shell *shell, size_t argc, char **argv)
 
 SHELL_CMD_ARG_REGISTER(mesh, &mesh_cmds, "Bluetooth mesh shell commands",
 			cmd_mesh, 1, 1);
+
+
+long bt_mesh_shell_strtol(const char *str, int base, int *err)
```

Comment:
Please move these 3 functions to `subsys/shell/shell_util.c`, that way other shell users can reuse them.